aemiers/rancid-tomatillos
# Feature/base api

## Pull Request

### Description

* Is this a feature or a fix?
This is a feature!

* Where should the reviewer start?
Please view the apiCalls.js file for the api calls. 
In App.js, please review the componentDidMount and clickHandler functions (where the API functions are invoked)

* What functionalities do these changes affect? Why were these changes made? What was the motivation behind these changes?
Changes the application from using fake data to data from the API.

* What outside features or research did you use to implement this fix?
stackoverflow! And the lessons from React

* How should this be tested?
Please pull down the branch locally and click through the application. Movies should be populating the main page and when a movie is clicked, the details should be expanded on that movie into a new screen. When you click the Rancid Tomatillos logo, you should be able to see the 'home' screen populated with movies again. 

***
closes #10 

#### Please review considerations below:
